app/src/main/java/org/jellyfin/androidtv/details/actions/ResumeAction.kt

```diff
@@ -0,0 +1,38 @@
+package org.jellyfin.androidtv.details.actions
+
+import android.content.Context
+import android.content.Intent
+import android.drm.DrmStore
+import android.util.Log
+import androidx.core.content.ContextCompat.startActivity
+import kotlinx.android.synthetic.main.text_under_button.view.*
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.TvApp
+import org.jellyfin.androidtv.playback.MediaManager
+import org.jellyfin.androidtv.util.TimeUtils
+import org.jellyfin.androidtv.util.apiclient.PlaybackHelper
+import org.jellyfin.apiclient.interaction.Response
+import org.jellyfin.apiclient.model.dto.BaseItemDto
+import org.jellyfin.apiclient.model.dto.BaseItemType
+
+private const val LOG_TAG = "ResumeAction"
+
+class ResumeAction(context: Context, val playbackPositionTicks: Long, val itemID: String) : PlaybackAction(ActionID.RESUME.id, context) {
+	val actualPlaybackPositionInMillis: Long
```

Comment:
```kotlin
val actualPlaybackPositionInMillis by lazy {
    playbackPositionTicks / 10000 - TvApp.getApplication().resumePreroll
}
```

The "by" syntax will move the setter & getter to a separate implementation. The "lazy" function will run the expression when the value of the property is first requested.

This would replace the set in the "init" function

Reference: https://kotlinlang.org/docs/reference/delegated-properties.html

Reply:
Ah, nice. Done.